test/cmocka/src/audio/selector/selector_test.c

```diff
@@ -438,8 +438,9 @@ static void test_audio_sel(void **state)
 		fill_source_s32(sel_state);
 		break;
 #endif /* CONFIG_FORMAT_S24LE || CONFIG_FORMAT_S32LE */
-/* TODO: add S24_3LE support */
-	case SOF_IPC_FRAME_S24_3LE:
```

Comment:
I understand this means no u8 and s24_3le support in cmocka, does s24_3le support in SOF for windows?  just want to double confirm.

Reply:
yes, it is supported by windows. SOF can support it but needs a lot of work.